Index: ibus-typing-booster/hunspell-tables/en_US.conf
===================================================================
--- ibus-typing-booster.orig/hunspell-tables/en_US.conf	2017-03-06 16:15:13.551991460 +0530
+++ ibus-typing-booster/hunspell-tables/en_US.conf	2017-03-06 16:15:13.547991387 +0530
@@ -8,7 +8,7 @@
 status_prompt = en
 layout = default
 hunspell_dict = en_US.dic
-hunspell_dict_package = hunspell-en
+hunspell_dict_package = hunspell-en-us
 ime_name = English - US (Typing Booster)
 symbol = en-US
 imes = Native Keyboard:NoIme,Latin-Pre:t-latn-pre,Latin-Post:t-latn-post
